app/services/API1811/FinancialTransactionsService.scala

```diff
@@ -29,19 +31,42 @@ import utils.LoggerUtil
 import scala.concurrent.{ExecutionContext, Future}
 
 class FinancialTransactionsService @Inject()(val connector: FinancialDataConnector,
+                                             val hipConnector: FinancialDataHIPConnector,
                                              implicit val ec: ExecutionContext) extends LoggerUtil {
   def getFinancialTransactions(regime: TaxRegime, queryParameters: FinancialRequestQueryParameters
-                              )(implicit headerCarrier: HeaderCarrier, appConfig: AppConfig, request: Request[_]): Future[FinancialTransactionsResponse] = {
-
-    logger.debug("[FinancialTransactionsService][getFinancialTransactions] " +
-      s"Calling financialDataConnector with Regime: $regime\nParams: $queryParameters")
-    connector.getFinancialData(regime, queryParameters).map {
-      case Right(financialTransactions) =>
-        infoLog(s"[FinancialTransactionsService][getFinancialTransactions] successfully retrieved financial transactions. Attempting to remove invalid charges")
-        Right(
-          financialTransactions.copy(documentDetails = ChargeTypes.removeInvalidCharges(financialTransactions.documentDetails))
-        )
-      case response: FinancialTransactionsResponse => response
+                              )(implicit headerCarrier: HeaderCarrier, appConfig: MicroserviceAppConfig, request: Request[_]): Future[FinancialTransactionsResponse] = {
+
+    if (appConfig.features.enable1811HIPCall()) {
+      logger.debug("[FinancialTransactionsService][getFinancialTransactions] - HIP Call enabled, calling FinancialDataHIPConnector")
```

Comment:
Please use logger.info instead of logger.debug as debug doesn't show up in kibana